src/style.css

```diff
@@ -1 +1,95 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-size: normal;
```

Comment:
The `font-size` property is set to 'normal', which is not a valid value. Please specify a valid size using units like `px`, `em`, or `rem`.